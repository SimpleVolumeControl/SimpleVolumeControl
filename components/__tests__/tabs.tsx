import { render } from '@testing-library/preact';
import Tabs from '../tabs';

describe('Tabs', () => {
  test(`should render correctly`, () => {
    const { asFragment } = render(
      <Tabs
        tabs={[
          { id: 0, title: 'a', content: 'A' },
          { id: 1, title: 'b', content: 'B' },
          { id: 2, title: 'c', content: 'C' },
        ]}
      />,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
