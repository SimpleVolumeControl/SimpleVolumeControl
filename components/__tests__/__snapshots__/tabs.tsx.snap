// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs should render correctly 1`] = `
<DocumentFragment>
  <div
    class="tabs -mb-px z-10"
  >
    <a
      class="tab md:tab-lg tab-lifted border-b-0 tab-active"
    >
      a
    </a>
    <a
      class="tab md:tab-lg tab-lifted border-b-0 "
    >
      b
    </a>
    <a
      class="tab md:tab-lg tab-lifted border-b-0 "
    >
      c
    </a>
    <span />
  </div>
  <div
    class="bg-base-100 border border-base-300 p-4 rounded-tr-box rounded-b-box"
  >
    A
  </div>
</DocumentFragment>
`;
